src/Symfony/Component/Form/ResolvedFormType.php

```diff
@@ -205,8 +205,30 @@ public function getOptionsResolver()
 
             $this->innerType->setDefaultOptions($this->optionsResolver);
 
+            $reflector = new \ReflectionMethod($this->innerType, 'setDefaultOptions');
+            $isOverwritten = ($reflector->getDeclaringClass()->getName() !== 'AbstractType');
```

Comment:
This will never work, because the class name is not AbstractType. So it will always be reported as overwritten